src/dotnet-install.ps1

```diff
@@ -871,7 +871,9 @@ if (!$isAssetInstalled) {
     $isAssetInstalled = Is-Dotnet-Package-Installed -InstallRoot $InstallRoot -RelativePathToPackage $dotnetPackageRelativePath -SpecificVersion $SpecificVersion
 }
 
+# Version verification failed. More likely something is wrong either with the downloaded content or with the verification algorithm.
 if (!$isAssetInstalled) {
+    Say-Error "Failed to verify the version of installed `"$assetName`", installation source: $DownloadLink. Report the bug at https://github.com/dotnet/install-scripts/issues."
```

Comment:
I would suggest to keep wording:
`If you think this is a bug, report it at https://github.com/dotnet/install-scripts/issues` as it might be also some user or user machine fault (though less chance)

Reply:
I thought about that too. I find the chances that it is a user / user machine fault quite low. Extraction did not fail, so it seems that something is wrong with the zip. On the contrary, the error you mention is a direct result of user input. User could easily make a typo in the `--arch` parameter and see the message. The situations are rather different. Let's hear other opinions on the matter.

Reply:
I suggest proceeding with the initial wording.